aio/content/marketing/announcements.json

```
@@ -1,7 +1,7 @@
 [
   {
-    "startDate": "2020-03-01",
-    "endDate": "2020-03-31",
+    "startDate": "2020-01-01",
+    "endDate": "2020-04-04",
```

We have a different notification for watching the live stream between April 1-3 (see below), so we probably shouldn't be showing that.

```suggestion
    "endDate": "2020-03-31",
```

---

I agree

---

@aaronfrost can you please update the PR with a fixup commit? thanks